src/cgame/cg_weapons.c

```diff
@@ -2351,6 +2379,257 @@ void CG_AddPlayerWeapon(refEntity_t *parent, playerState_t *ps, centity_t *cent)
+		break;
+	case WP_KAR98:
+	case WP_GPG40:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -4.0, 5.0, 1.5 },
+		                        (vec3_t) { 1.0, -2.0, 1.5 });
+		break;
+	case WP_THOMPSON:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -3.0, 3.5, 0.0 },
+		                        (vec3_t) { 2.5, -0.8, 0.5 });
+		break;
+	case WP_MP40:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -4.4, 5.0, 0.0 },
+		                        (vec3_t) { 1.0, 0.0, 0.0 });
+		break;
+	case WP_LUGER:
+	case WP_SILENCER:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { 0.0, 6.0, 2.0 },
+		                        (vec3_t) { 5.0, -3.0, 2.0 });
+		break;
+	case WP_COLT:
+	case WP_SILENCED_COLT:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -2.0, 6.0, 2.2 },
+		                        (vec3_t) { 3.0, -3.5, 2.2 });
+		break;
+	case WP_AKIMBO_LUGER:
+	case WP_AKIMBO_SILENCEDLUGER:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -3.0, 2.0, 0.0 },
+		                        (vec3_t) { 1.0, -3.0, 0.0 });
+		break;
+	case WP_AKIMBO_COLT:
+	case WP_AKIMBO_SILENCEDCOLT:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -2.5, 2.0, 0.0 },
+		                        (vec3_t) { 0.0, -3.0, 0.0 });
+		break;
+	case WP_KNIFE:
+	case WP_KNIFE_KABAR:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -4.5, 3.0, 0.0 },
+		                        (vec3_t) { 2.0, -4.0, 0.0 });
+		break;
+	case WP_LANDMINE:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -4.0, 6.0, 0.0 },
+		                        (vec3_t) { -2.0, 0.0, 0.0 });
+		break;
+	case WP_SATCHEL:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -3.0, 4.0, 0.0 },
+		                        (vec3_t) { 1.0, -5.0, 0.0 });
+		break;
+	case WP_SATCHEL_DET:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -3.0, 2.0, 0.0 },
+		                        (vec3_t) { 1.0, -5.0, 0.0 });
+		break;
+	case WP_DYNAMITE:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -1.0, 1.0, 0.0 },
+		                        (vec3_t) { 5.0, -7.0, 0.0 });
+		break;
+	case WP_MEDIC_SYRINGE:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -3.0, 0.0, 0.0 },
+		                        (vec3_t) { -4.0, 0.0, 0.0 });
+		break;
+	case WP_MEDIC_ADRENALINE:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -3.0, 5.0, 0.0 },
+		                        (vec3_t) { 3.0, 0.0, 2.0 });
+		break;
+	case WP_PLIERS:
+		CG_ApplyViewWeaponShift(hand,
+		                        (vec3_t) { -3.5, 2.0, 0.0 },
+		                        (vec3_t) { -7.8, 0.0, 6.0 });
+		break;
+	default:
+		// default to `cg_gunFov 75` - which was the universal default in 8.82.1
+		VectorMA(hand->origin, -3.0, up, hand->origin);
+		break;
+	}
+}
```

Comment:
This should be move in .weapon file as new field.

Reply:
> This should be move in .weapon file as new field.

Good point!